en/editor/project/index.md

```diff
@@ -59,6 +78,12 @@ Used to configure various parameters of physics, please refer to the [Physics Co
 
 ![scripting](./index/scripting.png)
 
+- **Conforming class fields**: if enabled, class fields are implemented using `Define` semantics; otherwise, class fields are implemented using `Set` semantics.
+
+- **Allows declaration of class fields**: if enabled, `declare` keyword may be used to declare class fields. In such case, class fields without explicit initializer would be initialized to undefined, according to the specification.
```

Comment:
```suggestion
- **Allows declaration of class fields**: if enabled, the `declare` keyword is allowed to declare class fields in TypeScript scripts. When a field is declared with `declare` and no explicit initialization is specified, it will be initialized to `undefined` according to the specification.
```